app/code/Magento/Directory/Model/Currency/Import/CurrencyConverterApi.php

```diff
@@ -12,7 +12,7 @@ class CurrencyConverterApi extends AbstractImport
     /**
      * @var string
      */
-    const CURRENCY_CONVERTER_URL = 'http://free.currencyconverterapi.com/api/v3/convert?q={{CURRENCY_FROM}}_{{CURRENCY_TO}}&compact=ultra'; //@codingStandardsIgnoreLine
+    const CURRENCY_CONVERTER_URL = 'http://free.currencyconverterapi.com/api/v6/convert?q={{CURRENCY_FROM}}_{{CURRENCY_TO}}&compact=ultra&apiKey={{API_KEY}}'; //@codingStandardsIgnoreLine
```

Comment:
What happened to V3 API?

Reply:
Both v3 and v6 require API Key, but since the latest version is v6, I changed it to v6.

Reply:
Then I suppose some functionality is broken, please report steps to reproduce as an issue and mention it in description.

Reply:
Let me add the steps in description now.

Reply:
@orlangur I just added the steps to reproduce the original error in description. Thanks!

Reply:
@Hailong please report an issue separately in https://github.com/magento/magento2/issues and then add it here to description

Reply:
Done.